app/jobs/reports/monthly_account_reuse_report.rb

```diff
@@ -0,0 +1,155 @@
+require 'csv'
+
+module Reports
+  class MonthlyAccountReuseReport < BaseReport
+    REPORT_NAME = 'monthly-account-reuse-report'
+
+    attr_reader :report_date
+
+    def perform(report_date)
+      @report_date = report_date
+
+      _latest, path = generate_s3_paths(REPORT_NAME, 'json', now: report_date)
+      body = report_body.to_json
+
+      [
+        bucket_name, # default reporting bucket
+        IdentityConfig.store.s3_public_reports_enabled && public_bucket_name,
+      ].select(&:present?).
+        each do |bucket_name|
+        upload_file_to_s3_bucket(
+          path: path,
+          body: body,
+          content_type: 'application/json',
+          bucket: bucket_name,
+        )
+      end
+    end
+
+    def first_day_of_report_month
+      report_date.beginning_of_month.strftime('%Y-%m-%d')
+    end
+
+    def params
+      {
+        query_date: first_day_of_report_month,
+      }.transform_values { |v| ActiveRecord::Base.connection.quote(v) }
+    end
+
+    def agency_reuse_results_json
+      agency_sql = format(<<-SQL, params)
+          SELECT
+              COUNT(*) AS num_users
+          , agencies_per_user.num_agencies
+          FROM (
+              SELECT
+                  COUNT(DISTINCT agencies.id) AS num_agencies
+              , identities.user_id
+              FROM 
+                  identities
+              JOIN 
+                  service_providers sp ON identities.service_provider = sp.issuer
+              JOIN 
+                  agencies ON sp.agency_id = agencies.id
+              WHERE
+                  identities.last_ial2_authenticated_at IS NOT NULL
+              AND
+                  identities.verified_at < %{query_date}
+              GROUP BY 
+                  identities.user_id
+          ) agencies_per_user
+          GROUP BY 
+              agencies_per_user.num_agencies
+          HAVING agencies_per_user.num_agencies > 1
+          ORDER BY 
+              num_agencies ASC
+      SQL
+
+      agency_results = transaction_with_timeout do
+        ActiveRecord::Base.connection.execute(agency_sql)
+      end
+
+      agency_results.as_json
+    end
+
+    def total_proofed_results
```

Comment:
I might call this `num_active_profiles` or something, to indicate that it's really just a single number instead of a whole array of rows/hashes like the others